### ISSUE TITLE
try CornerNet

### ISSUE THREAD

**Issue by tkng** [blue-oil/blueoil#71]
CornerNet is a recently proposed neural network for object detection.

http://openaccess.thecvf.com/content_ECCV_2018/html/Hei_Law_CornerNet_Detecting_Objects_ECCV_2018_paper.html

The idea of CornerNet looks very interesting. They don't use anchor boxes. They use 2 heatmaps to predict "top-left corner" and "bottom-right" corner. Though it looks a little bit difficult to implement on Blueoil, since it requires a novel operator "corner pooling", but at least it would be worth trying.

- [ ] implement corner pooling on lmnet
- [ ] implement hour glass network (backbone network)
- [ ] implement extraction network
- [ ] implement loss function of CornerNet on lmnet
- [ ] compare result vs YOLOv2 on Pascal VOC
- [ ] compare result vs YOLOv2 on MS COCO
- [ ] implement slimmer version of CornetNet
- [ ] implement corner pooling on dlk
- [ ] implement post processing

This task would be very interesting, but unfortunately I don't enough time to implement...

**Comment by tkng** [comment]
No worry, this issue is not high priotity for now.
